packages/cli/src/commands/deploy/render.js

```diff
@@ -0,0 +1,66 @@
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+
+export const command = 'render <side> [...commands]'
+export const description = 'Build command for Render deploy'
+export const builder = (yargs) => {
+  yargs
+    .positional('side', {
+      choices: ['api', 'web'],
+      description: 'select side to build',
+      type: 'string',
+    })
+    .option('build', {
+      description: 'Build for production',
+      type: 'boolean',
+      default: 'true',
+    })
+    .option('prisma', {
+      description: 'Apply database migrations',
+      type: 'boolean',
+      default: 'true',
+    })
+    .option('data-migrate', {
+      description: 'Migrate the data in your database',
+      type: 'boolean',
+      default: 'true',
+      alias: 'dm',
+    })
+    .epilogue(
+      `For more commands, options, and examples, see ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/docs/cli-commands#deploy'
+      )}`
+    )
+}
+
+export const handler = async ({ side, build, prisma, dm: dataMigrate }) => {
+  const paths = getPaths()
+  let commandSet = []
+  if (side == 'api') {
+    if (build) {
+      commandSet.push('yarn rw build api')
```

Comment:
per discussion with @SEANDOUGHTY need to include `yarn install` as first step

Reply:
Technically I think this command wouldn't have run if install hadn't run right?

Reply:
I added the `yarn` command to the actual build command. An additional `yarn install` here would be redundant.